### PR TITLE
[RISCV] Add `TuneVentanaVeyron` subtarget feature.

### PR DESCRIPTION
**Pull request by mgudim**
This will be used to add veyron fusions in a later commit.

### CODE REVIEW

File: llvm/lib/Target/RISCV/RISCVSubtarget.h [hunk 1]
```diff
@@ -46,10 +46,7 @@ struct RISCVTuneInfo {
 
 class RISCVSubtarget : public RISCVGenSubtargetInfo {
 public:
-  enum RISCVProcFamilyEnum : uint8_t {
-    Others,
-    SiFive7,
-  };
+  enum RISCVProcFamilyEnum : uint8_t { Others, SiFive7, VentanaVeyron };
```
Comment (topperc) [inline llvm/lib/Target/RISCV/RISCVSubtarget.h:49]: Please don't reformat this. I have other CPUs in this enum in my downstream. I'd prefer to keep one CPU per line. We can put a clang-format off/on around it.

Comment (mgudim) [reply llvm/lib/Target/RISCV/RISCVSubtarget.h]: sure.

File: llvm/lib/Target/RISCV/RISCVSubtarget.h [hunk 2]
```diff
@@ -46,11 +46,13 @@ struct RISCVTuneInfo {
 
 class RISCVSubtarget : public RISCVGenSubtargetInfo {
 public:
+  // clang-format off
   enum RISCVProcFamilyEnum : uint8_t {
     Others,
     SiFive7,
+    VentanaVeyron
```
Comment (topperc) [inline llvm/lib/Target/RISCV/RISCVSubtarget.h:53]: Leave a trailing comma after this. Makes it easier to insert new CPUs without touching this line.

Comment (mgudim) [reply llvm/lib/Target/RISCV/RISCVSubtarget.h]: done

**Review by topperc — APPROVED** [review]
LGTM

**Pull request merged**